README.md

```diff
@@ -1104,6 +1105,48 @@ bash -v scriptname
 bash -x scriptname
 ```
 
+# 5. Package Management
+There are two types of packages available for linux systems.
+* Debian Packages (.deb)
+* RedHat Package Manager (.rpm)
+
+## 5.1 Debian Package (deb)
+deb is the format, as well as extension of the software package format for the Debian distribution and its derivatives.
+
+### 5.1 A.`Update Packages`
+The Debian package manager updates its package data by ```bash sudo apt update```
+
+### 5.1 B.`Upgrade Packages`
+After updating package information if there is any update available then run ```bash sudo apt upgrade``` to get the available update.
+
+### 5.1 C.`Install Package`
+You can install new package by ```bash sudo apt install <package-name>```
+
+### 5.1 D.`Remove/Uninstall Package`
+To remove/uninstall package ```bash sudo apt autoremove <package-name>```
+
+### 5.1 E.`Install Required Dependecies`
```

Comment:
Maybe a little explanation on the close relation between `apt`, `apt-get`, and `dpkg` would be nice